packages/web-components/src/components/tile/__stories__/tile-group.stories.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit';
+import '../index';
+import readme from './README--tile-group.stories.mdx';
+import '@carbon/ibmdotcom-web-components/es/components/image';
+import {
+  Default as DefaultTile,
+  WithImage as TileWithImage,
+} from './tile.stories';
+import { CTA_TYPE } from '../../cta/defs';
+import { boolean, select, text } from '@storybook/addon-knobs';
+
+const ctaTypeOptions = Object.values(CTA_TYPE).filter((value) => !!value);
+
+export default {
+  title: 'Components/Tile Group',
+  parameters: {
+    ...readme.parameters,
+    hasStoryPadding: true,
+    knobs: () => {
```

Comment:
The knobs are still not showing up for the tile stories. Looking at other components, it looks like this property is expected to be an object keyed by the component, with a value of this function, eg.

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/src/components/quote/__stories__/quote.stories.ts#L64-L88